plugins/FeedReader/FeedReaderPlugin.cpp

```diff
@@ -55,7 +55,7 @@ extern "C" {
 #ifdef WIN32
 	__declspec(dllexport)
 #endif
-	uint32_t RETROSHARE_PLUGIN_revision = 0;
+	uint32_t RETROSHARE_PLUGIN_revision = abs(atoi(RS_EXTRA_VERSION)) ;
```

Comment:
abs() is here to replace a cast to uint32_t? Given that RS_EXTRA_VERSION has no reason to be negative, I would just do 
uint32_t RETROSHARE_PLUGIN_revision  = (uint32_t)atoi(RS_EXTRA_VERSION);

Reply:
changed it now